### DISCUSSION TITLE
HTTP Stufenschalter speichern nicht möglich nach dem zweiten Schalter

### DISCUSSION THREAD

**Discussion by Bernd275** [camueller/SmartApplianceEnabler#476] [Q&A]
Hallo, ich will 4 Shelly per Stufenschalter betreiben, sofern ich den zweiten Schalter hinzufügen möchte, wird der save Button aus gegraut und es ist kein speichern mehr möglich. Die SAE Version ist die 2.2.1

**Comment by camueller** [comment]
Es gab kürzlich eine ähnliche Fehlemeldung, die auf den verwendeten Browser zurückzuführen war (Safari).

Zuminest für Chrome, Firefox und Safari habe ich einen automatisierten Test, der auch einen Stufenschalter (allerdings mit GPIO statt HTTP) beinhaltet:
Siehe https://drive.google.com/file/d/1g7mgRbnyaPVroLKrlPBE5j1Jmu51WwqD/view bei Zeitstemp 5:48

Hast Du auch Leistungsstufen hinzugefügt? Ohne die geht es natürlich nicht ...

Kannst Du bitte testen, ob das Problem auch mit Chrome auftritt?

**Reply by Bernd275** [reply]
Hab eben mal Crome installiert und kann es da auch bestätigen, die erste Stufe wird gespeichert die zweite ist aus gegraut

![image](https://user-images.githubusercontent.com/122307990/219967548-aa669d12-256c-4124-9133-6f6edf8deed0.png)

![image](https://user-images.githubusercontent.com/122307990/219968368-63701d2e-bfff-482d-b037-6f7897671499.png)

![image](https://user-images.githubusercontent.com/122307990/219968313-02a1b2bd-3d96-41ca-8cc8-a5f471c398ae.png)

**Comment by camueller** [comment]
Da stimmt irgendetwas mit Deinem Browser nicht: Dein Screenhot zeigt zweimal "Leistungsstufen", davon eimal ohne "L". Auch bei Benachrichtigungen fehlt der erste Buchstabe.

**Reply by Bernd275** [reply]
das lag an mir, es waren drei Bilder.

![image](https://user-images.githubusercontent.com/122307990/219967894-d0cc3bd9-df72-4061-9a9a-014c587488d3.png)

**Comment by camueller** [comment]
Ich kann das Problem nicht nachstellen: Wenn ich einen Stufenschalter mit HTTP anlege mit 2 Leistungsstufen (alles entsprechend Deinem Screenshot) ist der Speichern-Button blau, d.h es kann gespeichert werden.

Offensichtlich machst Du irgendetwas anders. Kannst Du bitte eine genaue Liste mit allen Eingaben und Klicks aufschreiben.
Ich kann nur Fehler beheben, die ich reproduzieren kann.

**Reply by Bernd275** [reply]
Danke fürs Testen, ich habe immer ein und die selbe URL kopiert und bearbeitet, nachdem speichern von Schalter 1 hab ich dann den zweiten Schalter hinzufügen wollen. Das nimmt er mir dann nicht ab.
Ich hab es eben in einem Rutsch mit zwei Schaltern auch anlegen können, ein erweitern um einen dritten Schalter schlägt dann aber fehl.
Muss man die Stufenschalter in einem Rutsch anlegen?

**Reply by camueller** [reply]
Man kann nur gültige Konfigurationen speichern. Das muss nicht in einem Rutsch sein.

**Reply by Bernd275** [reply]
okay, ich kann es jetzt nicht erweitern um einen dritten Schalter.

![image](https://user-images.githubusercontent.com/122307990/219970883-6341c519-71ec-4600-ae0d-751ee81ec5aa.png)

![image](https://user-images.githubusercontent.com/122307990/219970358-80df9578-bb0b-4f82-b2dc-23e0a3b61b60.png)

**Comment by camueller** [comment]
Ich konnte es reproduzieren. Fix kommt mit der nächsten Bugfix-Version.

**Reply by Bernd275** [reply]
Vielen Dank für die Unterstützung
